homeassistant/components/tuya/cover.py

```diff
@@ -30,6 +30,25 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class _ControlValues(TypedDict):
```

Comment:
We do not need to introduce another level of definition, the entity description already does that. Let's just keep it at that level.

Reply:
I have now flattened it into the entity description. Please check again.